### ISSUE TITLE
links not highlighted in mu4e

### ISSUE THREAD

**Issue by dangom** [rougier/elegant-emacs#17]
In mu4e using the light theme, links are neither highlighted nor underscored

**Comment by rougier** [comment]
We disabled the underline but links should be highlighted (using the salient face). Could you post a screenshot?

**Comment by dangom** [comment]
Of course. This is an image were you don't see any links:

<img width="584" alt="screen-shot-invisible" src="https://user-images.githubusercontent.com/7881842/95783525-0437d280-0ca0-11eb-9b68-3092ec65ca17.png">

And when I hover the mouse over a sentence, suddenly you see it:

<img width="584" alt="screen-shot-visible" src="https://user-images.githubusercontent.com/7881842/95783570-23366480-0ca0-11eb-8a3d-5a70881d4ea3.png">

**Comment by rougier** [comment]
Can you go (with cursor) to the `voted into law` and type customize-face to check for the name of the face (and try to modify to check if it inherits from elegant-salient)?

**Comment by dangom** [comment]
No. I get suggestions for `variable-pitch, shr-link, mu4e-view-body-face`, none of which derive from elegant-salient. 
<img width="305" alt="image" src="https://user-images.githubusercontent.com/7881842/95870945-174bb080-0d3b-11eb-8d48-ac23da5b7505.png">

**Comment by rougier** [comment]
What is the face for `mu4e-link-face` ?

**Comment by dangom** [comment]
That one is indeed `face-salient`. Most links work. Only in some emails I have issues like the one above - when links are somehow extracted from html, but the description does not match the link itself.

**Comment by rougier** [comment]
Can you mail or forward me part of an email that is not working such that I can check?

**Comment by dangom** [comment]
Of course. I'm happy to. I'll send an email to the address at your profile.

**Comment by rougier** [comment]
Get the mail but I don't see links at all (not even regular one)
